pkgs/development/python-modules/pex/default.nix

```diff
@@ -8,14 +8,14 @@
 
 buildPythonPackage rec {
   pname = "pex";
-  version = "2.29.0";
+  version = "2.33.1";
   pyproject = true;
 
   disabled = pythonOlder "3.7";
 
   src = fetchPypi {
     inherit pname version;
-    hash = "sha256-2JeQ1zGKZorVqx+2OzjdF7nln00t42yxP2O0e6AVGx4=";
+    hash = "sha256-kHTvgXe51TencKDkFQAAdyPXuJLBNpJ0NIy1KB8p5JQ=";
   };
 
   build-system = [ hatchling ];
```

Comment:
The referenced PR migrated away from hatchling to setuptools, so this should be updated as well